NDTensors/ext/NDTEnsorsJLArraysExt/set_types.jl

```diff
@@ -0,0 +1,10 @@
+# TypeParameterAccessors definitions
+using NDTensors.TypeParameterAccessors: TypeParameterAccessors, Position
+using NDTensors.GPUArraysCoreExtensions: storagemode
+using JLArrays: JLArrays, JLArray
+
+function TypeParameterAccessors.default_type_parameters(::Type{<:JLArray})
+  return (Float64, 1)
+end
+TypeParameterAccessors.position(::Type{<:JLArray}, ::typeof(eltype)) = Position(1)
+TypeParameterAccessors.position(::Type{<:JLArray}, ::typeof(ndims)) = Position(2)
```

Comment:
These shouldn't be needed since I made generic definitions in #1505.

Reply:
Good point, I removed these definitions for all GPU backends. Thanks!